From 5c8a83710989a6c814c4538ef3b1a3bd7c379ce3 Mon Sep 17 00:00:00 2001
From: Edoardo Morandi <edoardo.morandi@m4ss.net>
Date: Wed, 4 Dec 2024 15:18:12 +0100
Subject: [PATCH] Configure for cross-compilation

Signed-off-by: Edoardo Morandi <edoardo.morandi@m4ss.net>
Upstream-Status: Pending
---
 script/probe.pl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/script/probe.pl b/script/probe.pl
index 1420de5..5ad10a2 100644
--- a/script/probe.pl
+++ b/script/probe.pl
@@ -25,6 +25,8 @@ my %status = (
 
 # These are based on experience in development of PkgConfig.pm:
 
+use Data::Dumper; print Dumper($ENV), '\n';
+
 $status{system_libdir}     = ['/usr/lib'];
 $status{system_includedir} = ['/usr/include'];
 
