From 5c8a83710989a6c814c4538ef3b1a3bd7c379ce3 Mon Sep 17 00:00:00 2001
From: Edoardo Morandi <edoardo.morandi@m4ss.net>
Date: Wed, 4 Dec 2024 15:18:12 +0100
Subject: [PATCH] Configure for cross-compilation

Signed-off-by: Edoardo Morandi <edoardo.morandi@m4ss.net>
Upstream-Status: Pending
---
 alienfile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/alienfile b/alienfile
index 18d6b42..5056d6e 100644
--- a/alienfile
+++ b/alienfile
@@ -20,6 +20,7 @@ share {
   plugin 'Build::Autoconf' => ();
 
   my $configure = '--disable-shared --enable-static --disable-builddir';
+  $configure .= " --host=$ENV{BUILD_SYS}" if $ENV{BUILD_SYS};
 
   if($^O eq 'MSWin32')
   {
