From 291cb4d86c274b090a714ed9c3c4132f17b1b188 Mon Sep 17 00:00:00 2001
From: Edoardo Morandi <edoardo.morandi@m4ss.net>
Date: Fri, 20 Dec 2024 16:28:57 +0100
Subject: [PATCH] use BUILD_SYS env as host if available

Signed-off-by: Edoardo Morandi <edoardo.morandi@m4ss.net>
Upstream-Status: Pending
---
 script/build.pl | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/script/build.pl b/script/build.pl
index 6a4bbac..84ca6d5 100644
--- a/script/build.pl
+++ b/script/build.pl
@@ -84,6 +84,12 @@ my @configure_flags = (
   '--with-system_includedir=' .  join($Config{path_sep}, @{ $probe->{system_includedir}  }),
 );
 
+if($ENV{HOST_SYS})
+{
+  unshift @configure_flags,
+      "--host=$ENV{HOST_SYS}";
+}
+
 if($type eq 'static')
 {
   unshift @configure_flags,
